Misc/NEWS.d/next/Library/2023-02-07-22-20-32.gh-issue-101362.Jlk6mt.rst

```diff
@@ -0,0 +1,3 @@
+Speed up :class:`pathlib.PurePath` construction by handling arguments more
+uniformly. When a path argument is supplied, we use its string
+representation rather than joining its parts with :func:`os.path.join`.
```

Comment:
```suggestion
uniformly. When a :class:`pathlib.Path argument is supplied,
we use its string representation rather than joining its parts
with :func:`os.path.join`.
```